lib/provider/map_notifier.dart

```diff
@@ -49,4 +65,15 @@ class MapNotifier with ChangeNotifier {
     final GoogleMapController controller = await _completer.future;
     controller.animateCamera(CameraUpdate.newCameraPosition(_currentPos));
   }
-}
+
+  Future<void> toggleCampus(LatLng latLng) async {
+    final c = await _completer.future;
+    final p = CameraPosition(
+      target: latLng,
+      zoom: constants.CAMERA_DEFAULT_ZOOM,
+      tilt: constants.CAMERA_DEFAULT_TILT,
+      bearing: 30.8334901395799,
```

Comment:
Why change the bearing from default on the toggle button? Seems like a strange and unwanted behavior.

Reply:
You're right this can automatically be done by a double swipe on the map so there is no need for it here.
Removed!